R-package/R/utils.R

```diff
@@ -374,10 +379,6 @@ NULL
 #' The \code{\link{xgb.save.raw}} function is useful if you'd like to persist the XGBoost model
 #' as part of another R object.
 #'
-#' Note: Do not use \code{\link{xgb.serialize}} to store models long-term. It persists not only the
-#' model but also internal configurations and parameters, and its format is not stable across
-#' multiple XGBoost versions. Use \code{\link{xgb.serialize}} only for checkpointing.
-#'
```

Comment:
What's the replacement for `xgb.serialize`? Should users use `saveRDS` to fully preserve all attributes?